build-test-batch.xml

```diff
@@ -2246,6 +2233,53 @@ information. Make sure to commit in all build services results.
 		</run-batch-test>
 	</target>
 
+	<target name="source-format-jdk8">
+		<run-batch-test>
+			<test-action>
+				<if>
+					<contains string="${liferay.releng.branch}" substring="master" />
```

Comment:
@Ithildir Since this is on the portal side, we are not 100% sure either but both Hashi and I think this should work.